code/src/java/pcgen/rules/persistence/token/PreCompatibilityToken.java

```diff
@@ -91,17 +91,17 @@ public String getTokenName()
 		return tokenName;
 	}
 
-	public int compatibilityLevel()
+	public static int compatibilityLevel()
```

Comment:
This method (and the two below) are used in the CompatibilityToken interface.
This is a red flag to me that something funny is going on and this check-in needs to be reviewed in more detail (there may be a code bug somewhere?)